addons/sale/wizard/sale_order_cancel.py

```diff
@@ -1,20 +1,87 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models
+from odoo import _, api, fields, models, tools
+from odoo.exceptions import UserError
 
 
 class SaleOrderCancel(models.TransientModel):
     _name = 'sale.order.cancel'
-    _description = "Sales Order Cancel"
+    _inherit = 'mail.composer.mixin'
+    _description = 'Sales Order Cancel'
 
+    @api.model
+    def _get_default_from(self):
```

Comment:
```suggestion
    def _default_email_from(self):
```

https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#symbols-and-conventions
-> Default method : the default method pattern is `_default_<field_name>`